src/frontend/components/SøknadsSteg/VelgBarn/LeggTilBarn/useNavnInputFelt.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
```

Comment:
Hvorfor valgte du å opprette en ny fil for denne hooken istedenfor å ha den sammen med useLeggTilBarn?

Reply:
Bra spørsmål :) egentlig bare for oversiktens og ryddighetens skyld. 

- at useLeggTilBarn ikke er både useLeggTilBarn og useNavnInput
- at filen ikke blir så stor
- at useNavnInput bare blir litt forstyrrende når man skal gjøre noe annet i useLeggTilBarn syntes jeg

innvendinger?

Reply:
Jeg bare merger, men du kan fortsatt komme med innvendinger om du vil altså 😊  det kan refaktoreres

Reply:
Neida, helt fair, bare lurte :)